blockstack/app.co
# [WIP] Referrals

This PR contains the implementation of Viral Loops, for tracking referrals to app mining.

TESTING:

- Visit https://staging.app.co
- Click 'invite' in bottom left
- Enter your info
- Get your referral link
- Visit the referral link
- Register an app **with a different email than the one you used to get a referral link**
- Go to https://app.viralloops.com , login with 1password
- Visit the 'testing' campaign
- See the full referral completed
